yarl/_url.py

```diff
@@ -382,10 +394,13 @@ def is_default_port(self):
         e.g. 'http://python.org' or 'http://python.org:80', False
         otherwise.
 
+        Return False for relative URLs
```

Comment:
Let's have a full sentence here, at least:
```suggestion
        Return False for relative URLs.
```